source/_integrations/apsystems.markdown

```diff
@@ -0,0 +1,25 @@
+---
+title: APsystems EZ1
+description: Control and monitor your APsystems EZ1 microinverters locally without the cloud
+featured: false
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_release: 2024.6
+ha_category:
+  - Energy
+ha_domain: apsystems
+ha_platforms:
+  - sensor
+ha_integration_type: device
```

Comment:
```suggestion
ha_integration_type: device
```
I tried to double-check for the integration type in the manifest, but could not find it. According to the [dev docs](https://developers.home-assistant.io/docs/creating_integration_manifest?_highlight=integration&_highlight=type#integration-type), it should be defined there.

Reply:
Created a PR: https://github.com/home-assistant/core/pull/117782